test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -349,6 +349,7 @@ public function returnTypeFunctionProvider()
             ['returnsString', 'string'],
             ['returnsNull', 'null'],
             ['returnsObject', '\stdClass'],
+            ['returnsVoid', 'void'],
```

Comment:
No, there's no method called that, that's just the test case name :P no magic here, just the `getTypeHint` method